sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/OrcFileFormat.scala

```diff
@@ -314,6 +314,7 @@ private[orc] object OrcFileFormat extends HiveInspectors with Logging {
     "NONE" -> "",
     "SNAPPY" -> ".snappy",
     "ZLIB" -> ".zlib",
+    "LZ4" -> ".lz4",
```

Comment:
Can you also add test for "hive" case?

Reply:
+1 for @viirya 's suggestion.

Reply:
Please add it to `HiveOrcSourceSuite.scala`.

Reply:
+1, I will add a test case